charts/castai-agent/templates/deployment.yaml

```diff
@@ -77,6 +77,18 @@ spec:
             - name: shared-metadata
               mountPath: /agent-metadata
           env:
+            - name: SELF_POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: SELF_POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: SELF_POD_NODE
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
```

Comment:
question: I understand why we need to pass namespace for this ticket, by why do we need to pass the pod and node?